posthog/test/base.py

```diff
@@ -458,6 +469,18 @@ def _create_person(*args, **kwargs):
     return Person(**{key: value for key, value in kwargs.items() if key != "distinct_ids"})
 
 
+def _create_session_recording_event(**kwargs):
+    """
+    Create a session recording event in tests. NOTE: all events get batched and only created when sync_execute is called
+    """
+    if not kwargs.get("uuid"):
+        kwargs["uuid"] = str(uuid.uuid4())
```

Comment:
you can streamline this and the one below with `kwargs.setdefault()`

Reply:
I'll treat this as a NIT and ignore unless a big objection!